tests/test_h5store.py

```diff
@@ -728,20 +728,22 @@ def test_single_writer_multiple_reader_different_process_swmr(self):
             raise
 
 
-@unittest.skipIf(not NUMPY, 'requires numpy package')
-@unittest.skipUnless(python_implementation() == 'CPython', 'Optimized for CPython.')
-class H5StorePerformanceTest(BaseH5StoreTest):
+@pytest.mark.skipif(not NUMPY,reason= 'requires numpy package')
+@pytest.mark.skipif(python_implementation() != 'CPython',reason= 'Optimized for CPython.')
+class TestH5StorePerformance(TestBaseH5Store):
     max_slowdown_vs_native_factor = 1.25
 
-    def setUp(self):
-        super(H5StorePerformanceTest, self).setUp()
+    @pytest.fixture
+    def setUp(self,setUp_base_h5Store):
```

Comment:
```suggestion
    def setUp(self, setUp_base_h5Store):
```
Please make sure to follow the formatting guidelines.